Signal/src/util/OWSBackup.m

```diff
@@ -483,6 +484,164 @@ - (void)clearAllCloudKitRecords
         }];
 }
 
+#pragma mark - Lazy Restore
+
+- (NSArray<NSString *> *)attachmentRecordNamesForLazyRestore
```

Comment:
When we "backup export", we need to preserve the records for all attachments that are lazy restoring.